apis/mattermost/v1alpha1/clusterinstallation_test.go

```diff
@@ -66,7 +66,7 @@ func TestClusterInstallation(t *testing.T) {
 			require.NoError(t, err)
 			assert.Equal(t, size1000.App.Replicas, tci.Spec.Replicas)
 			assert.Equal(t, size1000.App.Resources.String(), tci.Spec.Resources.String())
-			assert.Equal(t, size1000.Minio.Replicas, tci.Spec.Minio.Replicas)
+			assert.Equal(t, size1000.Minio.Servers, tci.Spec.Minio.Servers)
```

Comment:
Do we also need to check `VolumesPerServer`?

Reply:
You're totally right.